src/main/java/TaskManager.java

```diff
@@ -0,0 +1,55 @@
+public class TaskManager {
+
+    public static Task[] tasks = new Task[100];
+    public static int task_count = 0;
+
+    public static void handleCommand(String command){
+        if (command.matches("^list$")){
+            listTasks();
+        }
+        else if(command.matches("^mark [0-9]*$")){
+            int task_number = Integer.parseInt(command.substring(5)) - 1;
+            markTask(task_number);
+        }
+        else if (command.matches("^unmark [0-9]*$")){
+            int task_number = Integer.parseInt(command.substring(7)) - 1;
+            unmarkTask(task_number);
+        }
+        else{
+            Task task = new Task(command);
+            addTask(task);
+        }
```

Comment:
Perhaps move the `else if` and `else` blocks to start after the closing bracket (Cuddle braces).

Reply:
noted, will change and commit, thanks!